nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -0,0 +1,283 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  cfg = config.services.glitchtip;
+  pkg = cfg.package;
+  inherit (pkg.passthru) python;
+
+  manage = pkgs.writeShellScript "manage" ''
+    set -o allexport
+    ${lib.toShellVars environment}
+    ${lib.getExe pkgs.sudo} -E -u glitchtip ${lib.getExe pkg} "$@"
```

Comment:
We need to use the sudo wrapper under /run/wrappers otherwise the binary does not have the setuid bit

Reply:
Also we want to use cfg.user here